src/plugins/kconfig/README.md

```diff
@@ -0,0 +1,49 @@
+- infos = Information about the kconfig plugin is in keys below
+- infos/author = Dardan Haxhimustafa <mail@dardan.im>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/kconfig
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = recommended maintained compatible specific experimental unfinished nodoc concept
+- infos/metadata =
+- infos/description = Reads and writes the KConfig ini format
+
+## Introduction
+
+This plugin can be used to parse a KConfig ini file into a KeySet and save a given KeySet to such a file.
+
+Information about the syntax:
+
+- Comments start with character `#`
+- Keys can contain spaces and special characters except from the main special characters (`[`, `]`, `=`, `#`and `$`)
+- If a key has values, then it will be followed with an `=` symbol and then the value will be read to the end of the line
+- Can contain multiple keys with different locales (`keyName[en]` and `keyName[de]`)
+- Cannot contain multiple keys with different metadata (either `keyname[$a]` or `keyname[$b]`).
+- If a key `keyname[$metavalue]` is parsed, it will be represented as a Key with name `parent/keyname` and meta `kconfig` as `metavalue`
```

Comment:
it is allowed to have multiple meta values for a key (not for groups, for keys only), but not same as in locales. An example would be `key.name[$a][$i] = Key Value`. 
This would be saved in kdb with `... KEY_META, "kconfig", "ai", KEY_END);`
Because the meta keys are only allowed to be one character (as far as I understood, even one byte, not really utf-8 character), I just put the metas `a` and `i` together in a string as `ai`.

This will likely change in the PR for `https://github.com/ElektraInitiative/kconfig/issues/10` though

Reply:
If it will change, then do not write so much about it (or only as limitation)